conf/jvm-clients.options

```diff
@@ -1,10 +1,11 @@
 ###########################################################################
 #                         jvm-clients.options                             #
 #                                                                         #
-# See jvm8-clients.options and jvm11-clients.options for Java version     #
-# specific options.                                                       #
+# See jvm11-clients.options, jvm17-clients.options, and                   #
+# jvm21-clients.options for Java version specific options.                #
```

Comment:
This is interesting... we technically add to HCD the config files for JDK22 but we do not advertise we support it. Not sure how that will be handled